app/views/teachers/components/LicenseStatsModal.vue

```diff
@@ -0,0 +1,84 @@
+<script>
+const moment = require('moment')
+export default Vue.extend({
+  name: 'LicenseStatsModal',
+  props: ['hide', 'loading', 'redeemers', 'removedRedeemers'],
+  computed: {
+    moment () {
+      return moment
+    }
+  }
+})
+
+</script>
+
+<template lang="pug">
+#modal-base-flat
```

Comment:
can't we use Modal component from `app/components/common/Modal` instead?

Reply:
ok we have a new modal in `app/components/common` :joy: i believe i used it once but i totally forget it

Reply:
oh but i don't make `modal` display normally. seems it is transparency at all. so i still using the old modal 😂